tools/crm_resource.c

```diff
@@ -910,6 +912,11 @@ validate_or_force_cb(const gchar *option_name, const gchar *optarg,
     if (options.override_params == NULL) {
         options.override_params = pcmk__strkey_table(free, free);
     }
+
+    if (optarg != NULL) {
+        options.check_level = atoi(optarg);
```

Comment:
It's better to use pcmk__parse_min_int() to get error checking (we can warn user and ignore). atoi() will just return 0 if an invalid value is given

Reply:
Fixed, and also updated the help output.